src/TouristInfoCards.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+
+function TouristInfoCards() {
+  return (
+    <div>
```

Comment:
You might want to add an attribute `className="tourist-cards-container"` here and use css to centre the 3 cards. GIve it a go! 🙂
```suggestion
    <div className="tourist-cards-container">
```